src/web/mjs/engine/ComicInfoGenerator.mjs

```diff
@@ -0,0 +1,29 @@
+export default class ComicInfoGenerator {
```

Comment:
This class only contains `static` methods.
Consider to use instance methods, or remove the class and just export the public methods from this module file.

Reply:
I fixed the code by changing it from a static method to an instance method, similar to how other classes, like BookmarkManager, work